engine/src/juliabox/db/instance.py

```diff
@@ -99,7 +99,12 @@ def get_stale_instances(cluster):
         valid_time = nowsecs - JBoxInstanceProps.SESS_UPDATE_INTERVAL
         stale = []
         for record in JBoxInstanceProps.scan(instance_id__beginswith=cluster, publish_time__lt=valid_time):
```

Comment:
Also need the same change in `get_instance_status` and `get_available_instances` below.